src/images/maps/geom_adm0.R

```diff
@@ -25,6 +40,75 @@ geom_adm0 <- function(iso3) {
       call. = FALSE
     )
   }
-
   gg$geom_sf(data = sf_adm0)
 }
+
+#' assert_within_distance
+#' Assert that `x` is within `dist` distance from `y`.
+#' @param x sf class with geometry
+#' @param y sf class with geometry
+#' @param dist distance in meters (default = 10000)
+#'
+#' @return TRUE if all elements of x fall within `dist` distance of y.
+#'     Return error if any elements fall outside of distance
+#' @export
+#'
+#' @examples
+#' library(sf)
```

Comment:
These examples won't work because you have no actual box imports. I think we can keep them but just note testing will be a much more complex setup.

Reply:
yeah ok. I like have the examples there as you can easily run through them (even in roxygen format!) manually and see the functions in action.

But yeah when it comes using for actual testing/CI im not sure how it fits into this repo design/box etc. I had just taken use of `{box}` out of the roxygen examples based on this comment: 
https://github.com/OCHA-DAP/hdx-signals/pull/75#discussion_r1598621531

but for the manual testing benefit box vs no box doesnt matter much